src/Sylius/Bundle/AttributeBundle/Form/EventSubscriber/BuildAttributeValueFormSubscriber.php

```diff
@@ -11,6 +11,7 @@
 
 namespace Sylius\Bundle\AttributeBundle\Form\EventSubscriber;
 
+use Sylius\Bundle\AttributeBundle\Form\Type\AttributeValueType\Configuration\Factory\AttributeValueTypeConfigurationFactoryInterface;
```

Comment:
`AttributeValueType\Configuration\Factory\AttributeValueTypeConfigurationFactoryInterface` - it looks like `AttributeValueTypeConfigurationFactory` is doubled here